src/ui/server/controllers/insurance/business/company-details/index.test.ts

```diff
@@ -49,20 +55,44 @@ describe('controllers/insurance/business/companies-details', () => {
   });
 
   describe('pageVariables', () => {
-    it('should have correct properties', () => {
-      const result = pageVariables(mockApplication.referenceNumber);
-
-      const expected = {
-        POST_ROUTES: {
-          COMPANIES_HOUSE: `${INSURANCE_ROOT}/${mockApplication.referenceNumber}${COMPANY_HOUSE_SEARCH}`,
-          COMPANY_DETAILS: `${INSURANCE_ROOT}/${mockApplication.referenceNumber}${COMPANY_DETAILS_ROUTE}`,
-          NO_COMPANIES_HOUSE_NUMBER: `${INSURANCE_ROOT}/${mockApplication.referenceNumber}${NO_COMPANIES_HOUSE_NUMBER}`,
-          SAVE_AND_BACK_URL: `${INSURANCE_ROOT}/${mockApplication.referenceNumber}${COMPANY_DETAILS_SAVE_AND_BACK}`,
-        },
-        FIELDS: EXPORTER_BUSINESS,
-      };
-
-      expect(result).toEqual(expected);
+    describe('originalUrl does not contain "change"', () => {
```

Comment:
Not a blocker for this PR, but can we align with the other descriptions, it's not "contain" 😄